tests/test_tensorflow.py

```diff
@@ -110,41 +105,29 @@ def multioutput(self, x, z):
     )
 
 
-class TestTensorFlow(TestCase):
-
-    maxDiff = 4096
-
-    def setUp(self):
-        # Clear existing model
-        if os.path.isdir(tf_export_path):
-            shutil.rmtree(tf_export_path)
-
-    def tearDown(self):
-        self.setUp()
-
-    def test_tensorflow(self):
-        if tf.__version__ < '2':
-            _make_model_v1()
-        else:
-            _make_model_v2()
-
-        # Make the model description
-        metadata = TensorFlowModel.create_model(tf_export_path).set_name('tf').set_title('TF')
-
-        # Make the servable
-        model = TensorFlowServable(**metadata.to_dict())
-
-        # Test it out
-        print(model.run([[2, 3, 4]]))
-        self.assertEqual((1, 3), np.shape(model.run([[2, 3, 4]])))
-        self.assertTrue(np.isclose([[3, 4, 5]], model.run([[2, 3, 4]])).all())
-        self.assertTrue(np.isclose([[4, 8, 12]],
-                                   model.scalar_multiply(([[1, 2, 3]], 4))).all())
-        self.assertAlmostEqual(3, model.length([[1, 2, 3]]))
-
-        # Test the multioutput
-        output = model.multioutput(([[1, 2, 3]], 4))
-        self.assertIsInstance(output, list)
-        self.assertEqual(2, len(output))
-        self.assertEqual((1, 3), np.shape(output[0]))
-        self.assertEqual((1, 3), np.shape(output[1]))
+def test_tensorflow(tmpdir):
```

Comment:
is `tmpdir` defined somewhere? curious how one runs this

Reply:
As we discussed on Friday and I place for record here, it's [part of pytest](https://docs.pytest.org/en/6.2.x/tmpdir.html)